pkg/gds/emails/sentlogs.go

```diff
@@ -0,0 +1,37 @@
+// This contains helper functions to help confirm if emails were sent.
+package emails
+
+import (
+	"time"
+
+	"github.com/trisacrypto/directory/pkg/gds/models/v1"
+	pb "github.com/trisacrypto/trisa/pkg/trisa/gds/models/v1beta1"
+)
+
+// VerifyEmailSentVASPContact checks if an email has been sent to the VASP contacts in some time window (e.g. the past 30 days).
+// Returns false for either email has not been sent or an error occurred.
+func VerifyEmailSentVASPContact(vasp *pb.VASP, kind string, timeWindowDays int) (status bool, err error) {
+
+	contact := models.ContactFromType(vasp.Contacts, kind)
+	emailLog, err := models.GetEmailLog(contact)
+	if err != nil {
+		status = false
```

Comment:
Note: Go assigns default values to declared variables (false for bool), so it's not necessary to set status to false since it's declared in the returned params. This also works for ints (initialized to 0) and slices (initialized to nil).